connect/src/com/telenor/mobileconnect/SimCardStateChangedBroadcastReceiver.java

```diff
@@ -0,0 +1,35 @@
+package com.telenor.mobileconnect;
+
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.content.IntentFilter;
+
+import com.telenor.connect.ConnectSdk;
+
+public class SimCardStateChangedBroadcastReceiver extends BroadcastReceiver {
+
+    private static SimCardStateChangedBroadcastReceiver sSimCardChangedReceiver =
+            new SimCardStateChangedBroadcastReceiver();
+
+    private static IntentFilter SIM_CARD_CHANGED_INTENT_FILTER =
+            new IntentFilter("android.intent.action.SIM_STATE_CHANGED");
```

Comment:
Seems this intent isn't officially documented, so I would maybe add a comment, possibly linking to http://stackoverflow.com/questions/10528464/how-to-monitor-sim-state-change , for future reference.